docker/statsd/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012 Chatham Financial Corp <oss@chathamfinancial.com>
```

Comment:
Is this license somehow required?

Reply:
I believe so, yes. Since I used files directly from the repo in question, I didn't want to leave the license out.